pkg/cmd/server/admin/create_nodeconfig.go

```diff
@@ -86,6 +88,7 @@ func NewCommandNodeConfig(commandName string, fullName string, out io.Writer) *c
 	flags.StringVar(&options.DNSDomain, "dns-domain", options.DNSDomain, "DNS domain for the cluster.")
 	flags.StringVar(&options.DNSIP, "dns-ip", options.DNSIP, "DNS server IP for the cluster.")
 	flags.Var(&options.ListenAddr, "listen", "The address to listen for connections on (scheme://host:port).")
+	flags.DurationVar(&options.SyncPeriod, "iptables-sync-period", options.SyncPeriod, "How often iptables rules are refreshed (e.g. '5s', '1m', '2h22m').  Must be greater than 0.")
```

Comment:
this is an advanced setting... we should probably expose it in the config, but not in the launch args